ci/windows/build_common.psm1

```diff
@@ -1,4 +1,4 @@
-Param(
```

Comment:
Diffs on these lines are due to BOM updates explicitly telling powershell that these are UTF-8 encoded.